example/README

```diff
@@ -98,3 +98,19 @@ Restrictions:
 set OFP_CONFIG to OFP_CONFIG_NETWRAP_WEBSERVER
 e.g.
 #define OFP_CONFIG OFP_CONFIG_NETWRAP_WEBSERVER
+
+Environment considerations
+===============================================================================
```

Comment:
I'd prefer that this section is at the top of the file so it's harder to miss by a user.

Reply:
ok